720:Updated the map and map background for better visibility in dark mode

osamaahmed17 opened the pull request:
Resolves #720 
I removed all the changes that I made in the previous commits and just added the dark background for dark mode.

https://github.com/user-attachments/assets/5d1da340-9a71-4656-8516-33f72804eb2e

arkid15r reviewed (commented):
Please link an issue to the PR.

osamaahmed17 commented:
ok

osamaahmed17 commented:
@aramattamara, feel free to give your review on this pull request.

kasya reviewed (changes requested):
@osamaahmed17 hey! I'm sorry, but this is not what the issue was about at all. 
The problem was with the background in dark theme being too light. 
After looking into this some more I realized that it would be better to remove the `.leaflet-container` class  that does background and outline-offset completely 🤔
![Screenshot 2025-02-15 at 2 30 41 PM](https://github.com/user-attachments/assets/c4d1bb37-1711-4465-aa03-58986790293c)

I'd like to keep the style as it was before in a dark mode.

osamaahmed17 commented:
@arkid15r, I added the requested change, but it is still failing some checks.

osamaahmed17 commented:
I removed all the changes that I made in the previous commits and just added the dark background for dark mode.
https://github.com/user-attachments/assets/3cfc5070-56d2-4e70-a793-523fa1805433

kasya reviewed (approved):
LGTM! Thanks!
